### ISSUE TITLE
Changes for PayPal affect core config fields with tooltip 

### ISSUE THREAD

**Issue by AndreasMueller75** [magento/magento2#21974]
### Preconditions (*)

1. Magento 2.3.1
2. PHP 7.*
### Steps to reproduce (*)

1. create an config field, type **`time`** with a tooltip in a group in a section (see below)
```
<?xml version="1.0" ?>
<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
    <system>
        <section id="shipping" showInDefault="1" showInStore="1" showInWebsite="1" translate="label tooltip comment">
            <group id="test_group" showInDefault="1" showInStore="1" showInWebsite="1" sortOrder="0" translate="label tooltip comment">
                <field id="test_time" showInDefault="1" showInWebsite="1" showInStore="0" sortOrder="10"
                       translate="label tooltip comment" type="time">
                   <label><![CDATA[Test Time]]></label>
                   <source_model>Magento\Config\Model\Config\Source\Date\Short</source_model>
                   <tooltip><![CDATA[Testtooltip.]]></tooltip>
                </field>
            </group>
        </section>
    </system>
</config>

```

### Expected result (*)

The time fields are separated by colons (:)
<img width="731" alt="expected" src="https://user-images.githubusercontent.com/32193606/55164450-fbd68000-516b-11e9-9572-9cb29a8bd67e.png">

### Actual result (*)

The colons are missing (have a font-size of 0).
<img width="752" alt="actual" src="https://user-images.githubusercontent.com/32193606/55164538-1b6da880-516c-11e9-8c0c-232530c180f0.png">

### The reason

The change in vendor/magento/theme-adminhtml-backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less from [the following commit](https://github.com/magento/magento2/commit/ab2c789a251154ad8df703c6c4132c95915a3f4b#diff-dd4d98e90feb13843af90622ac96a666R495) sets the font-size of all input-field labels with a tooltip to 0.

```
 .value {
            padding-right: 40px;
            vertical-align: middle;
            width: 44%;

            &.with-tooltip {
                font-size: 0;
```

### Proposed Resolution
The css selector (`.accordion .config .value.with-tooltip`) is too general and should be more precise for setting font-size: 0 in PayPals usecase.

**Comment by magento-engcom-team** [comment]
Hi @AndreasMueller75. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@AndreasMueller75 do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by AndreasMueller75** [comment]
Hi, i reworked the issue description. The issue is easily reproducible in Magento 2.3.1.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-backlog-nazar 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-98971` were created

**Issue Available**: @engcom-backlog-nazar, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by geet07** [comment]
I am working on this at #dmcdindia19”

**Comment by magento-engcom-team** [comment]
@geet07 thank you for joining. Please accept team invitation [here](https://github.com/orgs/magento/invitation) and self-assign the issue.

**Comment by magento-engcom-team** [comment]
Hi @AndreasMueller75. Thank you for your report. 
The issue has been fixed in magento/magento2#23393 by @geet07 in 2.3-develop branch
Related commit(s):
 - [eb28f14cc4a1d8b5c3d5ac5bff037a3b5661b833](https://github.com/magento/magento2/commit/eb28f14cc4a1d8b5c3d5ac5bff037a3b5661b833)

The fix will be available with the upcoming 2.3.3 release.
